examples/xgboost/agaricus.ipynb

```diff
@@ -82,7 +82,136 @@
    "source": [
     "test_runner = bentoml.xgboost.get(\"agaricus:latest\").to_runner()\n",
     "test_runner.init_local()\n",
-    "test_runner.run([[0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 1, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1]])  # => array(0.01241208, dtype=float32)"
+    "test_runner.run(\n",
```

Comment:
does our black script format notebooks? this looks really bad in notebooks, we should not do that